_summaries/sutskever2014sequence_1.md

```diff
@@ -1,19 +1,48 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: stskever2014sequence
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: saikm200022
+score: 9
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+# **Summary - Sequence to Sequence Learning with Neural Networks, Sutskever, Vinyals, Le; 2014**
+
+## What is the core idea?
+
+Deep Neural Networks are being used today for all kinds of difficult tasks such as image detection and speech recognition. However, DNNs are not suitable for sequence to sequence tasks because the dimensionality of input and output sequences can be arbitrary. This paper suggests a Deep Learning approach that is able to apply DNNs for a sequence to sequence task by bypassing its limitation of requiring fixed dimensionality input and output.
+
+![Alt Text](sutskever2014sequence_1b.PNG)
+
+
+## How is it realized (technically)?
+
+The model was used on the WMT'14 English to French Machine Translation task. The paper trained the model on a subset of 12 million sentences with vocabulary size of 348 million and 304 million words for French and English respectively. 
+
+The model in the paper uses two LSTMs: one for the input and output sequences respectively. These LSTMs were relatively deep with four layers and a final softmax layer over the LSTMs output. The paper's training algorithm maximized the following to obtain the most likely translation: 
+
+![Alt Text](sutskever2014sequence_1a.PNG)
+
```

Comment:
Before elaborating about the final decoding details with beam search, it would be helpful to give more context about this MLE goal, like the model is predicting the probability distribution over vocabularies at each timestep. So that the transition to beam search would make more sense. Also, it should be noted that final decoding is the last post-processing step which doesn't get involved in training the MLE goal.

Reply:
I have added the suggested points.